code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -635,3 +755,18 @@ proc/chemical_mob_spawn(var/datum/reagents/holder, var/amount_to_spawn, var/reac
 	id = "mutadone"
 	description = "Chance to remove genetic disabilities."
 	color = "#C8A5DC" // rgb: 200, 165, 220
+
+datum/reagent/rejuvenators
+	name = "Rejuvenators"
+	id = "rejuvenators"
+	description = "100% chance per cycle of healing 2 points of BRUTE and BURN.."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+
+datum/reagent/rejuvenators/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustBruteLoss(-2*REM)
+	M.adjustFireLoss(-2*REM)
+	..()
+	return
+
```

Comment:
Pretty OP; Goonchem specifically avoids per cycle multiheals because they're mindless--this is effective better than old tricord (and omnizine) considering these are the two most common damage types.